changelogs/unreleased/gh-9806-http-set-version.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/http client
+
+* Added an option to set http version.
```

Comment:
Please, be certain: 'added the XXX option to the YYY function' would be better. Also, please, add an issue number (see other changelog entries for examples).

Reply:
Fixed

Reply:
I'll quote and comment.

> * Added the http-version option to the
>   `require()` function from `http` module (gh-9806)

1. `require()` -> `request()` (and `post()`, `get()`, etc)
2. `http-version` -> `http_version`
3. Unusual formatting: the line break after ~40 characters, not close to 80 (use `gqap` if you're using Vim).
4. Add a period a the end.

Reply:
Fixed

Reply:
5. `http` module. (gh-xxxx) -> `http.client` module (gh-xxxx).

(The module name and the period position.)

Sorry, overlooked it on the first glance.

Reply:
Fixed

Reply:
Sorry for nitpicking, but, please, adjust the period position as well.

```diff
 * Added the `http_version` option to the `request()` (and `post()`,
-  `get()`, etc) functions from `http.client` module. (gh-9806)
+  `get()`, etc) functions from `http.client` module (gh-9806).
```

(Also, I think that it should be 'the `http.client` module', with the definite article.)

Reply:
Fixed)